plugin-server/src/worker/ingestion/event-pipeline/runAsyncHandlersStep.ts

```diff
@@ -9,13 +9,21 @@ import { EventPipelineRunner } from './runner'
 export async function processOnEventStep(runner: EventPipelineRunner, event: PostIngestionEvent) {
     const processedPluginEvent = convertToProcessedPluginEvent(event)
 
-    await runInstrumentedFunction({
-        event: processedPluginEvent,
-        func: (event) => runOnEvent(runner.hub, event),
-        statsKey: `kafka_queue.single_on_event`,
-        timeoutMessage: `After 30 seconds still running onEvent`,
-        teamId: event.teamId,
-    })
+    const timeout = setTimeout(() => {
+        console.log(`⌛⌛⌛ Still running onEvent for team ${event.teamId}`)
+    }, 30 * 1000) // 30 seconds
+
+    try {
+        await runInstrumentedFunction({
+            event: processedPluginEvent,
+            func: (event) => runOnEvent(runner.hub, event),
+            statsKey: `kafka_queue.single_on_event`,
+            timeoutMessage: `After 30 seconds still running onEvent`,
```

Comment:
question: Can't we just add `${event.teamId}` to this timeout message?

Reply:
I mean instead of creating a new timeout.

Reply:
not if we don't want it to show up in sentry like that - https://github.com/PostHog/posthog/blob/05079fa1a786afbc3688f8f290374b8d90909fd7/plugin-server/src/utils/db/utils.ts#L38 which I'd argue we don't

Reply:
Ah, that's fine 👍 Wasn't aware about this showing up in sentry.